addons/binding/org.openhab.binding.onebusaway/src/main/java/org/openhab/binding/onebusaway/internal/handler/StopHandler.java

```diff
@@ -55,21 +56,20 @@
 
     public static final ThingTypeUID SUPPORTED_THING_TYPE = THING_TYPE_STOP;
 
+    private final Logger logger = LoggerFactory.getLogger(StopHandler.class);
+
     private StopConfiguration config;
     private Gson gson;
     private HttpClient httpClient;
-    private Logger logger = LoggerFactory.getLogger(StopHandler.class);
     private ScheduledFuture<?> pollingJob;
-    private Runnable pollingRunnable = () -> {
-        fetchAndUpdateStopData();
-    };
     private AtomicBoolean fetchInProgress = new AtomicBoolean(false);
     private long routeDataLastUpdateMs = 0;
-    private Multimap<String, ObaStopArrivalResponse.ArrivalAndDeparture> routeData = ArrayListMultimap.create();
+    private Map<String, List<ObaStopArrivalResponse.ArrivalAndDeparture>> routeData = new HashMap<>();
```

Comment:
This could be a `final` property. We have to change line 210 to `routeData.clear()` for that.